src/mainboard/pcengines/apu2/romstage.c

```diff
@@ -95,6 +96,18 @@ void cache_as_ram_main(unsigned long bist, unsigned long cpu_init_detectedx)
 
 		console_init();
 
+		nuvoton_assign_io_port_gpio(GPIO_CONFIG_DEV, 0x220);
```

Comment:
@piotr-kleins why all these configuration? Entry in devicetree is enough and proper handling on superio driverside.

Reply:
That config set appropiate registers in Nuvoton. Need to be done to enable direct gpio access.

Reply:
No it doesn't. The ramstage driver for Nuvoton will do this according to devicetree. I have explained that to you how it works...

Reply:
Okay, I've got it.But there must be set also LDN8 CR30.1 bit to 1. I believe it can't be done another way then writting it manually?

Reply:
`device pnp 2e.8 on` -> the **on** option will do that for you